package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-easy-crop",
-  "version": "3.0.1",
+  "version": "3.1.0",
```

Comment:
You should not update the version number in the PR. This will done on `master` by using `np`.

Reply:
I suppose it isn't a problem for this PR because the version between the local package and the dependency in docs was already different, but when working with different versions I could only get yarn to pick up the change when the version number changed. Should I also revert the lockfile change (32566a5539a12edf2843575b4b6873c131982679) and then open another PR after publishing to update the docs with the new version then?

Reply:
Yes, updating the version in the doc is quite annoying but is done a manual commit after the new version is released.

The real version of the package should not be increased in PR, otherwise, it'll prevent me from publishing the new version with the default pipeline.

Reply:
👍 done!